curriculum/challenges/english/09-information-security/information-security-projects/stock-price-checker.english.md

```diff
@@ -9,10 +9,10 @@ forumTopicId: 301572
 
 ## Description
 <section id='description'>
-Build a full stack JavaScript app that is functionally similar to this: <a href='https://sphenoid-crater.glitch.me/' target='_blank'>https://sphenoid-crater.glitch.me/</a>.
-Since all reliable stock price APIs require an API key, we've built a workaround. Use <a href='https://repeated-alpaca.glitch.me/' target='_blank'>https://repeated-alpaca.glitch.me/</a> to get up-to-date stock price information without needing to sign up for your own key.
-Working on this project will involve you writing your code on Glitch on our starter project. After completing this project you can copy your public glitch url (to the homepage of your app) into this screen to test it! Optionally you may choose to write your project on another platform but must be publicly visible for our testing.
-Start this project on Glitch using <a href='https://glitch.com/edit/#!/remix/clone-from-repo?REPO_URL=https://github.com/freeCodeCamp/boilerplate-project-stockchecker/'>this link</a> or clone <a href='https://github.com/freeCodeCamp/boilerplate-project-stockchecker/'>this repository</a> on GitHub! If you use Glitch, remember to save the link to your project somewhere safe!
+Build a full stack JavaScript app that is functionally similar to this: <a href="https://stock-price-checker--freecodecamp.repl.co/" target="_blank">https://stock-price-checker--freecodecamp.repl.co/</a>.
+Since all reliable stock price APIs require an API key, we've built a workaround. Use <a href="https://Stock-Price-Checker-Proxy--freecodecamp.repl.co" target="_blank">https://Stock-Price-Checker-Proxy--freecodecamp.repl.co</a> to get up-to-date stock price information without needing to sign up for your own key.
```

Comment:
```suggestion
Since all reliable stock price APIs require an API key, we've built a workaround. Use <a href="https://stock-price-checker-proxy--freecodecamp.repl.co" target="_blank">https://stock-price-checker-proxy--freecodecamp.repl.co</a> to get up-to-date stock price information without needing to sign up for your own key.
```
Nitpicks